res/controllers/Pioneer-DDJ-400-script.js

```diff
@@ -265,10 +265,14 @@ PioneerDDJ400.beatFxRightPressed = function(_channel, _control, value) {
 
 PioneerDDJ400.beatFxSelectPressed = function(_channel, _control, value) {
     if (value === 0) { return; }
-
-    engine.setValue("[EffectRack1_EffectUnit1]", "focused_effect", 3);
+    var focusfx = engine.getValue("[EffectRack1_EffectUnit1]", "focused_effect")+1;
+    if (engine.getValue("[EffectRack1_EffectUnit1]", "focused_effect")===3){
```

Comment:
Please add a short comment what this code does. I guess cycling through the 2 focused effects 1 and 2?

Please also check the coding style, i.e. add spaces and indentation.

Reply:
I commented everything now, my first contrib on GitHub, sorry.